### PR TITLE
Support EU Cloud Region

### PR DESCRIPTION
**Pull request by cea2aj**
Add a cloud region config option to the global_config to allow either 'us' (default), or 'eu' to be set as the cloud region.

The cloud region determines where the search assets and ytag are sourced from.

J=BACK-2277
TEST=manual, auto

Using the test site, set the cloudRegion to "eu" and confirmed the assets are loaded from the EU CDN and executed searches against the EU search endpoints

### CODE REVIEW

File: hbshelpers/sdkAssetUrl.js [hunk 1]
```diff
@@ -24,17 +30,21 @@ module.exports = function sdkAssetUrl(branch, locale, assetName) {
...
+    (isReleasedBranch || isPreReleaseBranch || isI18nFeatureBranch || isDevelopBranch) &&
     !(locale.startsWith('en') || assetName === 'answers.css') ;
-  
+
   const parsedAssetName = isLocalizationSupported ?
     `${locale}-${assetName}` :
     assetName;
 
-  return `https://assets.sitescdn.net/answers/${parsedBranch}/${parsedAssetName}`;
+  const domain = cloudRegion === 'eu'
```
Comment (EmilyZhang777) [inline hbshelpers/sdkAssetUrl.js:45]: can you create a const for 'eu' to improve readability? Maybe one for 'us' as well.

File: test-site/config/global_config.json [hunk 2]
```diff
@@ -1,9 +1,10 @@
 {
-  "sdkVersion": "develop", // The version of the Answers SDK to use
+  "sdkVersion": "1.15", // The version of the Answers SDK to use
```
Comment (EmilyZhang777) [inline test-site/config/global_config.json:2]: is this change intended?

**Review by EmilyZhang777 — COMMENTED** [review]
Only some nit.

File: layouts/html.hbs [hunk 3]
```diff
@@ -24,8 +24,12 @@
     <link rel="dns-prefetch" href="//dynl.mktgcdn.com">
     <link rel="dns-prefetch" href="//dynm.mktgcdn.com">
```
Comment (benmcginnis) [inline layouts/html.hbs:25]: do we know if these domains will be different @EmilyZhang777 ?

**Review by benmcginnis — APPROVED** [review]
LGTM, just had one question but it can be handled later if there is a change we have to make.

File: hbshelpers/sdkAssetUrl.js (hunk 1 above)
Comment (cea2aj) [reply hbshelpers/sdkAssetUrl.js]: sure

File: test-site/config/global_config.json (hunk 2 above)
Comment (cea2aj) [reply test-site/config/global_config.json]: nope, reverted

**Review by benmcginnis — APPROVED** [review]

File: layouts/html.hbs (hunk 3 above)
Comment (EmilyZhang777) [reply layouts/html.hbs:25]: I'm not sure. Based on [this](https://yext.slack.com/archives/CNVUY09HS/p1679068294340899) conversation, seems like eu is going to another bucket?

**Review by EmilyZhang777 — APPROVED** [review]
Only a small comment that we can address later. Otherwise, lgtm!
